partners.json

```
@@ -845,5 +845,26 @@
         "height": 154,
         "width": 100,
         "alt": "Logo du partenaire Communauté d’Agglomération du Bassin de Brive"
+    },
+    {
+        "name": "Communauté d’agglomération de Saint-Quentin-en-Yvelines",
+        "link": "https://www.saint-quentin-en-yvelines.fr/fr",
+        "infos": "La Direction des Systèmes d’Information de Saint-Quentin-en-Yvelines accompagne les 12 communes de l’agglomération sur un outil mutualisé et assure la publication de leurs Bases Adresses Locales.",
+        "perimeter": "Communes de l’agglomération de Saint-Quentin-en-Yvelines",
+        "codeDepartement": [
+            "95"
```

Le partenaire ne remonte pas dans les recherches concernant ses communes. Il me semble que le code département renseigné n'est pas le bon.

```suggestion
            "78"
```